lib/parity/backup.rb

```diff
@@ -53,10 +53,15 @@ def download_remote_backup
     def restore_from_local_temp_backup
       Kernel.system(
         "pg_restore tmp/latest.backup --verbose --clean --no-acl --no-owner "\
-          "-d #{development_db} #{additional_args}",
+          "--dbname #{development_db} --jobs `#{number_of_cores}` "\
+          "#{additional_args}",
       )
     end
 
+    def number_of_cores
+      "getconf _NPROCESSORS_ONLN"
+    end
```

Comment:
First: this is a really cool idea. Second: I'm really glad people with more shell-fu than I have area commenting.

Third (and less fun): I'd like to avoid any external gem dependencies for the time being. With Traveling Ruby being unwilling to move on from their current version of Bundler, we cause a permissions error on some OS versions when we try to install dependencies. I have plans to move away from Traveling Ruby, but that's not a short-term plan.